DICOM/Dicom-Anonymization.sln

```diff
@@ -0,0 +1,60 @@
+﻿
```

Comment:
Maybe we can name it Dicom.Anonymizer to be consistent with Fhir.Anonymizer?

Reply:
Thanks, changed it.